### PR TITLE
[Avada] - Added support for translating Circles info widgets

### PR DESCRIPTION
**Pull request by andrescifuentesr**
- Added support for translating Circles info widgets
- https://onthegosystems.myjetbrains.com/youtrack/issue/compsupp-7530/Avada-Circle-Info-Widget-not-translatable

### CODE REVIEW

File: avada/wpml-config.xml [hunk 1]
```diff
@@ -816,5 +816,13 @@
         <shortcode>
             <tag>fusion_form_submit</tag>
         </shortcode>
+        <shortcode>
+            <tag>fusion_circle_info</tag>
+            <attributes>
+                <attribute label="Circles Info - Title">title</attribute>
```
Comment (dgwatkins) [inline avada/wpml-config.xml:822]: Please use `:` as a separator @andrescifuentesr

Comment (dgwatkins) [reply avada/wpml-config.xml:822]: You could optionally just remove the label. 

We will automatically remove the `fusion_` prefix and generate "Circle Info: Title", so we don't really need a specific label here @andrescifuentesr

Comment (andrescifuentesr) [reply avada/wpml-config.xml]: Thanks @dgwatkins, automatically adding those labels seems a nice move. 
I just sent a new commit.

**Review by dgwatkins — APPROVED** [review]
👍  @andrescifuentesr

**Comment by dgwatkins** [comment]
merged and deployed @andrescifuentesr

**Pull request merged**